README.md

```diff
@@ -109,7 +109,7 @@ $ docker build -t my-app .
 $ docker run -p 3000:3000 --volume 'pwd':/usr/src/app --network --env-file .env my-app
 
 # run using docker compose
-$ docker-compose up
+$ docker compose up
```

Comment:
```suggestion
$ docker-compose up
```
Has the command changed?

Reply:
Docker now includes compose as a built-in command